docs/anatomy/anatomy-releases.md

```diff
@@ -67,6 +67,27 @@ Follow the below steps to tag your Release:
 
 ![Confirm Tag](/img/anatomy/confirm-tag.png)
 
+### Promote a Release
+
+KintoHub provides you an option to `Promote` your Releases across the different environments.
+
+You can do the following to **Promote** the release:
+
+1. Click on the `...` in the row of the `Tagged` release you wish to promote.
+2. Click on the **Promote Tag to Env** option
+
+![Promote Env](/img/anatomy/promote-env.png)
+
+3. Select the environment from the list you wish to promote the release in the prompt displayed.
```

Comment:
Action : change sentence structure => In the displayed prompt, select the environment you wish to promote the release to.

Reply:
Done